### ISSUE TITLE
Please consider adding ISBNs to booksdb.js

### ISSUE THREAD

**Issue by thenku** [ibm-developer-skills-network/rpszc-nodejsFinalModule#219]
The assignment requires matching ISBNs but none are present. ;-)

**Comment by Israel-Arendse** [comment]
@thenku  I agree, there were none in my project either but I managed to add my isbn properties with some custom number values from this free app. https://generate.plus/en/number/isbn.

As for how I wrote the ISBN property, the code format looks like this:

`1:{"isbn":"isbn-value", "author": "name", "title": "Book title", "reviews":{ user}} `

**Comment by ManavShah2400** [comment]
"1" is itself a ISBN just in different format

**Comment by Israel-Arendse** [comment]
@ManavShah2400 Really, I didn't know about that? Can  you please give me an example?

**Comment by mythemedesign** [comment]
use numbers 1 to 10 for ISBN.

**Comment by Navaneeth-21** [comment]
1 to 10 numbers are considered as ISBN. I hope this will help you
